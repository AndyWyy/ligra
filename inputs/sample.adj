WeightedAdjacencyGraph
4
6
0
2
4
5
0
2
3
1
0
2
3
6
1
5
4
5
