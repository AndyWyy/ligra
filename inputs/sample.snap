0 0 3
0 2 6
1 3 1
1 1 5
2 0 4
3 2 5